# E3.1

In [23]:
mu = 3000
x = c(3003,3005,2997,3006,2999,2998,3007,3005,3001)
n = length(x)

s_mean = mean(x)
"Sample mean:";  s_mean

var_sum = 0
for (e in x) {
    var_sum = var_sum + (e-s_mean)^2
}
s_var = var_sum/(n-1)
s_sd = sqrt(s_var)
"Sample Sd:"; s_sd
sd(x)

s_error = s_sd/sqrt(n)
"Standard error of the mean:"; s_error

[1] "Sample mean:"

[1] 3002.333

[1] "Sample Sd:"

[1] 3.708099

[1] 3.708099

[1] "Standard error of the mean:"

[1] 1.236033

**Intepretation:**  
It means that the mean of the sample is 3002 and has a standard deviation of 3.7 which means most items will be within $\pm$ 3.7*3 of the mean.
The standard error gives the average difference between the sample mean and the true mean, which in this case is approx 1.24.
The difference betwwen sample standard deviation and standard error, is that sd determines the range which the true mean is in, and standard error says that with the sample size of 9, the sample mean on average be 1.24 from true mean.

In [22]:
# b)
sigma = 3
mu = 3000

## Simulation
k = 10000
x = replicate(5, rnorm(k, mean = mu, sd = sigma))
head(x)

y = rowSums(x)
head(y)
mean(abs(y - y_mu) > 10)

## Theory
y_mu = mu*5
y_sd = sqrt(sigma^2*5)

2*(1-pnorm(y_mu+10, y_mu, y_sd))


[,1]     [,2]     [,3]     [,4]     [,5]    
[1,] 3002.966 2997.791 3001.011 3000.327 2999.681
[2,] 3003.926 3001.126 3001.852 2996.543 3000.745
[3,] 3001.107 3003.216 3003.077 2997.960 2996.269
[4,] 3002.621 3000.786 2999.498 2999.405 2999.534
[5,] 2998.064 2997.930 3003.023 2998.330 2999.532
[6,] 3002.751 3002.122 3002.718 3000.017 2997.924

[1] 15001.78 15004.19 15001.63 15001.84 14996.88 15005.53

[1] 0.1426

[1] 0.1360371

which means 13 - 14 percent of the time the construction is beyond 1cm of the target.

In [31]:
# c)
alpha = 0.05
tval = qt(1-alpha/2, n-1)

s_mean + c(-1,1)*tval*s_error

## R
t.test(x)


[1] 2999.483 3005.184


	One Sample t-test

data:  x
t = 2429, df = 8, p-value < 2.2e-16
alternative hypothesis: true mean is not equal to 0
95 percent confidence interval:
 2999.483 3005.184
sample estimates:
mean of x 
 3002.333 


In [36]:
# d)
alpha = 0.01
tval = qt(1-alpha/2, n-1)

s_mean + c(-1,1)*tval*s_error

## R
#t.test(x, conf.level = 0.99)

[1] 2998.186 3006.481

As we need to be more confident in our interval the higher confidence interval has a broader range.

In [45]:
# e)
alpha = 0.05
cvals = qchisq(p=c(1-alpha/2, alpha/2), df=n-1)
cvals

var_range = (n-1)*s_var/cvals
var_range

sd_range = sqrt(var_range)
sd_range

[1] 17.534546  2.179731

[1]  6.27333 50.46495

[1] 2.504662 7.103869

In [46]:
# f)
alpha = 0.01
cvals = qchisq(p=c(1-alpha/2, alpha/2), df=n-1)
cvals

var_range = (n-1)*s_var/cvals
var_range

sd_range = sqrt(var_range)
sd_range

[1] 21.954955  1.344413

[1]  5.010259 81.820090

[1] 2.238361 9.045446

# E3.2

In [48]:
x = c(180.02, 180.00, 180.01, 179.97, 179.92, 180.05, 179.94, 180.10,
180.24, 180.12, 180.13, 180.22, 179.96, 180.10, 179.96, 180.06)
n = 16

# a
s_mu = 180.05
s_sd = 0.0959

sd_error = s_sd/sqrt(n)

alpha = 0.1
c_val = qt(1-alpha/2,df = n-1)

s_mu + c(-1,1)*c_val*sd_error

## or
t.test(x, conf.level= 0.9)

[1] 180.008 180.092


	One Sample t-test

data:  x
t = 7508.6, df = 15, p-value < 2.2e-16
alternative hypothesis: true mean is not equal to 0
90 percent confidence interval:
 180.008 180.092
sample estimates:
mean of x 
   180.05 


In [51]:
# b)
alpha = 0.01
c_vals = qchisq(p=c(1-alpha/2, alpha/2), df=n-1)

var_range = (n-1)*s_sd^2/c_vals
var_range

sd_range = sqrt(var_range)
sd_range

[1] 0.004205689 0.029983630

[1] 0.06485128 0.17315782

# E3.3

In [55]:
mu = 3000
x = c(3003,3005,2997,3006,2999,2998,3007,3005,3001)
n = length(x)

# a)
t.test(x, mu = mu)


	One Sample t-test

data:  x
t = 1.8878, df = 8, p-value = 0.09576
alternative hypothesis: true mean is not equal to 3000
95 percent confidence interval:
 2999.483 3005.184
sample estimates:
mean of x 
 3002.333 


There's weak evidence against the null hypothesis, it's therefore likely that the true mean is actually 3000.

In [56]:
# b)
qt(0.995, df = n-1)

[1] 3.355387

In [58]:
# c)
alpha = 0.05
c(-1,1)*qt(1-alpha/2, df = n-1)


[1] -2.306004  2.306004

For a lower confidence level, we have a smaller range which makes sense

# E3.4

In [59]:
x = c(180.02, 180.00, 180.01, 179.97, 179.92, 180.05, 179.94, 180.10,
180.24, 180.12, 180.13, 180.22, 179.96, 180.10, 179.96, 180.06)
n = length(x)

# a)
t.test(x,mu = 180)


	One Sample t-test

data:  x
t = 2.0851, df = 15, p-value = 0.05456
alternative hypothesis: true mean is not equal to 180
95 percent confidence interval:
 179.9989 180.1011
sample estimates:
mean of x 
   180.05 


In [60]:
# b)
pt(0.995,df = n-1)

[1] 0.8322414

In [61]:
# c)
t.test(x, mu = 180, conf.level = 0.99)


	One Sample t-test

data:  x
t = 2.0851, df = 15, p-value = 0.05456
alternative hypothesis: true mean is not equal to 180
99 percent confidence interval:
 179.9793 180.1207
sample estimates:
mean of x 
   180.05 


In [62]:
# d)
t.test(x, mu = 180)


	One Sample t-test

data:  x
t = 2.0851, df = 15, p-value = 0.05456
alternative hypothesis: true mean is not equal to 180
95 percent confidence interval:
 179.9989 180.1011
sample estimates:
mean of x 
   180.05 


# Test 5

In [7]:
# 1
s_mu = 44.6
s = 0.424
mu = 45
n = 6

t_obs = (s_mu-mu)/(s/sqrt(n))
t_obs
pval = 2*(1-pt(abs(t_obs), df=n-1))
pval

[1] -2.310839

[1] 0.06883517

In [8]:
# 2
n = 20
theta = 1/4
a = 6

dbinom(6, 20, 1/4)

[1] 0.1686093

In [15]:
# 3
k = 1000
x = replicate(10, 2)
x = append(x, replicate(25, 4))
x = append(x, replicate(30,7))
x = append(x, replicate(25,10))
x = append(x, replicate(10,12))

var(x)

[1] 9.59596

In [20]:
# 4
n = 48
s_mu = 1.363
s = 1.521

tobs = (s_mu - 0)/(s/sqrt(n))
tobs
qt(0.975,df=n-1)

[1] 6.208508

[1] 2.011741

[1] 1.300532e-07

In [ ]:
# 5
